data/autoyast_qam/15-common_base_installation.xml.ep

```diff
@@ -313,7 +313,7 @@
   <users config:type="list">
     <user>
       <encrypted config:type="boolean">false</encrypted>
-      <fullname>Bernhard M. Wiedemann</fullname>
+      <fullname>bernhard</fullname>
```

Comment:
Can you also verify on some other server role?

Reply:
[mru-install-minimal-with-addons](https://openqa.suse.de/tests/overview?arch=&flavor=&machine=&test=mru-install-minimal-with-addons&modules=&module_re=&distri=sle&build=20230504-1&groupid=414#)

Reply:
[gnome-install](http://openqa.suse.de/t11042504)